### PR TITLE
Add TinkerHost Free SSL Verification

### PR DESCRIPTION
**Pull request by greenreader9**
Adds Domain Connect support to TinkerHost's existing SSL tool

Note to maintainers: The underscore on line 15 is intentional

### CODE REVIEW

**Comment by kerolasa** [comment]
Would it be possible to drop commits f9413e3 and 26ab6a4 from this pull.

File: tinkerhost.net.ggssl-verification.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "version": 1,
+  "syncPubKeyDomain": "domainconnect.tinkerhost.net",
+  "syncRedirectDomain": "app.tinkerhost.net",
+  "logoUrl": "https://app.tinkerhost.net/assets/image/th.svg",
+  "description": "Verifies a domain name with our SSL provider, GoGetSSL",
+  "variableDescription": "host describes the subdomain at which the verification key is avaiable. destination describes the second part of the verification key",
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "_%host%",
```
Comment (pawel-kow) [inline tinkerhost.net.ggssl-verification.json:15]: I don't think this usage of special variable %host% is intended here.

Please take a look at the output from the [online editor](https://pdnsadmin.revproxy.short-lived.de/dc/free/templateedit)

image

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: tinkerhost.net.ggssl-verification.json (hunk 1 above)
Comment (greenreader9) [reply tinkerhost.net.ggssl-verification.json]: Fixed in [3500533](https://github.com/Domain-Connect/Templates/pull/566/commits/35005332f5fe7e001aeeb94e99e54a0e1ee38a45)

**Review by pawel-kow — APPROVED** [review]
lgtm

**Pull request merged**